releases/Process/Major Release Process.md

```diff
@@ -59,91 +58,117 @@ The release process is managed by sig-release. To request an update to this docu
 Each of the below phases is represented on the Github Issues project board for the release. 
 
 ## Phase (1 of 4): Planning
-*Intent: Identify all stakeholders and their requirements for the release. Confirm stakeholder responsibilities.* 
-Begins 3 months prior to estimated release.
-Entry Criteria: sig-release has has discussed the need for a release, per current schedule. 
+*Intent: Identify all stakeholders and their requirements for the release. Confirm stakeholder responsibilities.
+Begins: 3 months prior to estimated release.
```

Comment:
@tonybalandiuk I think it will be useful to mention this in this step so the RM can use it as a reference on the release dates: https://www.o3de.org/docs/release-notes/archive/. Thoughts?

Reply:
Thanks, I added it!